index.html

```diff
@@ -15,5 +15,7 @@ <h1 class="headline">Your Money Management Assistant</h1>
       <button class="sign-in btn">Sign In</button>
       <button class="sign-up btn">Sign Up</button>
     </article>
+
+    <script src="./script/script.js"></script>
```

Comment:
remove scripts directory, and keep only file script.js, so your connection should look like src='./script.js'

Reply:
But you said that literally i must have 2 folder: style and script. Now should I remove script folder?